st2client/tests/unit/test_commands.py

```diff
@@ -73,22 +73,8 @@ def test_command_get_by_id(self):
         self.assertEqual(actual, expected)
 
     @mock.patch.object(
-        models.ResourceManager, 'get_by_name',
-        mock.MagicMock(return_value=base.FakeResource(**base.RESOURCES[0])))
-    @mock.patch.object(
-        models.ResourceManager, 'get_by_id',
-        mock.MagicMock(return_value=None))
-    def test_command_get_by_name(self):
```

Comment:
I removed this bogus test. The underlying code was anyways calling get_by_id and we are already testing that in previous test case.